Assets/Scripts/Models/Furniture.cs

```diff
@@ -132,9 +132,9 @@ private Furniture(Furniture other)
         UnlocalizedDescription = other.UnlocalizedDescription;
     }
 
-    public event Action<Furniture> Changed;
+    public event Action<Furniture> OnChanged;
 
-    public event Action<Furniture> Removed;
+    public event Action<Furniture> OnRemoved;
```

Comment:
No not know why people think that On... is good name for event. OnChadged is good name for method attached to event.